### PR TITLE
Update vhm-vmware.adoc

### PR DESCRIPTION
**Pull request by coryrwagner**
I couldn't find the "Download trusted root CA certificates" link, but I found article 2108294, which provided help. Downloading directly to the SUMA server is easier. The example for step 3 makes renaming and copying the files easy. Hopefully this will help someone else.

# Description

Short summary of why you created this PR

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [ ] Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: modules/client-configuration/pages/vhm-vmware.adoc [hunk 1]
```diff
@@ -67,14 +67,24 @@ If you see SSL errors while configuring your VMWare installation, you need to do
...
+done
+----
 . Change the filename suffix on the copied certificate to either ``.crt`` or ``.pem``.
 . On the {productname} Server, at the command prompt, update the CA certificate record:
 +
 ----
 update-ca-certificates
 ----
+
+For more information on how to download and install vCenter Server root certificates, see the VMWare article 2108294: https://kb.vmware.com/s/article/2108294
```
Comment (Loquacity) [inline modules/client-configuration/pages/vhm-vmware.adoc:90]:
```suggestion
For more information on how to download and install vCenter Server root certificates, see: https://kb.vmware.com/s/article/2108294
```

**Review by Loquacity — COMMENTED** [review]
Suggestions for style. Thanks for this PR!

**Review by Loquacity — APPROVED** [review]

File: modules/client-configuration/pages/vhm-vmware.adoc [hunk 2]
```diff
@@ -67,14 +67,30 @@ If you see SSL errors while configuring your VMWare installation, you need to do
 
 .Procedure: Trusting VMWare CA Certificates
 . Download the CA Certificate from your VMWare installation.
-    You can do this by logging in to your vCenter {webui}, and clicking btn:[Download trusted root CA certificates].
+    You can do this from the {productname} Server with this command:
++
+----
+wget --no-check-certificate https://vcenter.domain.com/certs/download.zip
```
Comment (keichwa) [inline modules/client-configuration/pages/vhm-vmware.adoc:73]:
```suggestion
wget --no-check-certificate https://vcenter.example.com/certs/download.zip
```

File: modules/client-configuration/pages/vhm-vmware.adoc [hunk 3]
```diff
@@ -67,14 +67,30 @@ If you see SSL errors while configuring your VMWare installation, you need to do
...
+----
++
+Replace ``vcenter.domain.com`` with the fully-qualified hostname for your VMWare installation.
```
Comment (keichwa) [inline modules/client-configuration/pages/vhm-vmware.adoc:76]:
```suggestion
Replace ``vcenter.example.com`` with the fully qualified hostname for your VMWare installation.
```

File: modules/client-configuration/pages/vhm-vmware.adoc [hunk 4]
```diff
@@ -67,14 +67,30 @@ If you see SSL errors while configuring your VMWare installation, you need to do
...
+done
+----
 . Change the filename suffix on the copied certificate to either ``.crt`` or ``.pem``.
 . On the {productname} Server, at the command prompt, update the CA certificate record:
 +
 ----
 update-ca-certificates
 ----
+
+For more information on how to download and install vCenter Server root certificates, see: https://kb.vmware.com/s/article/2108294
```
Comment (keichwa) [inline modules/client-configuration/pages/vhm-vmware.adoc:96]:
```suggestion
For more information on how to download and install vCenter Server root certificates, see: https://kb.vmware.com/s/article/2108294.
```

**Review by keichwa — APPROVED** [review]
minor cosmetic changes for consistency.

**Pull request merged**